src/components/filters.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+    <nav class="navbar navbar-expand-lg navbar-light justify-content-between mb-5">
+        <h5 class="m-0">filters</h5>
+        <div class="collapse navbar-collapse" id="navbarNavAltMarkup">
+            <div class="navbar-nav">
+                <a class="nav-item nav-link disabled">gender</a>
+                <a class="nav-item nav-link" @click="filterGender(genderVal)" v-for="genderVal in genders" :key="genderVal">
+                    <span :class="isActive(genderVal)">{{ genderVal }}</span>
+                </a>
+                <a class="nav-item nav-link disabled">no. of results</a>
+                <a class="nav-item nav-link" @click="filterNoOfResults(noOfResultsVal)" v-for="noOfResultsVal in nosOfResults" :key="noOfResultsVal">
+                    <span :class="isActive(noOfResultsVal)">{{ noOfResultsVal }}</span>
+                </a>
+                <a class="nav-item nav-link active" @click="filterReset()">reset</a>
+            </div>
+        </div>
+    </nav>
+</template>
+
+<script lang="ts">
+import { defineComponent, ref } from 'vue'
+
+export default defineComponent({
+    name: 'filters',
+    props: {
+        gender: {
+            type: String,
+            required: true
+        },
+        noOfResults: {
+            type: Number,
+            required: true
+        }
+    },
+    setup(props, { emit }) {
+        const currentGender = ref(props.gender)
```

Comment:
noted! but I added the currentGender with the props.gender as its ref since it will be updated later on
![image](https://user-images.githubusercontent.com/92789332/144011077-49ef390e-27d2-49b7-931c-c14ceae9e9a5.png)